tests/validation/tests/v3_api/common.py

```diff
@@ -478,8 +485,15 @@ def validate_workload(p_client, workload, type, ns_name, pod_count=1,
         pods = p_client.list_pod(workloadId=workload.id).data
         assert len(pods) == pod_count
     for pod in pods:
-        p = wait_for_pod_to_running(p_client, pod)
-        assert p["status"]["phase"] == "Running"
+        if type == "job":
+            job_type = True
+            expected_status = "Succeeded"
+        else:
+            job_type = False
+            expected_status = "Running"
+        p = wait_for_pod_to_running(p_client, pod, job_type=job_type)
+        assert p["status"]["phase"] == expected_status
+
 
```

Comment:
All the states for active pods deployment,daemon set are "Running" but for jobs the status is "Succeeded". therefore added a parameter job_type where the status is checked for Succeeded only if its a job.